src/ErrorHandler.php

```diff
@@ -357,18 +357,19 @@ public function addExceptionHandlerListener(callable $listener): void
      * Handles errors by capturing them through the Raven client according to
      * the configured bit field.
      *
-     * @param int    $level   The level of the error raised, represented by one
-     *                        of the E_* constants
-     * @param string $message The error message
-     * @param string $file    The filename the error was raised in
-     * @param int    $line    The line number the error was raised at
+     * @param int    $level      The level of the error raised, represented by
+     *                           one of the E_* constants
+     * @param string $message    The error message
+     * @param string $file       The filename the error was raised in
+     * @param int    $line       The line number the error was raised at
+     * @param array  $errcontext The error context (deprecated in PHP 7.2)
      *
      * @return bool If the function returns `false` then the PHP native error
      *              handler will be called
      *
      * @throws \Throwable
      */
-    private function handleError(int $level, string $message, string $file, int $line): bool
+    private function handleError(int $level, string $message, string $file, int $line, array $errcontext): bool
```

Comment:
According to the documentation the fifth parameter is optional, so add a default value for it (empty array)